README.md

```diff
@@ -3,8 +3,9 @@
 <br /><br />
 <a href="https://badge.fury.io/py/dagster"><img src="https://badge.fury.io/py/dagster.svg"></>
 <a href="https://coveralls.io/github/dagster-io/dagster?branch=master"><img src="https://coveralls.io/repos/github/dagster-io/dagster/badge.svg?branch=master"></a>
-<a href="https://buildkite.com/dagster/dagster"><img src="https://badge.buildkite.com/888545beab829e41e5d7303db15525a2bc3b0f0e33a72759ac.svg?branch=master"></a>
-<a href="https://dagster-slackin.herokuapp.com/"><img src="https://dagster-slackin.herokuapp.com/badge.svg"></a>
+<a href="https://twitter.com/dagsterio"><img src="https://img.shields.io/twitter/follow/dagsterio"></a>
+<a href="https://dagster.io/slack"><img src="https://dagster-slackin.herokuapp.com/badge.svg"></a>
+<a href="https://github.com/dagster-io/dagster"><img src="https://img.shields.io/github/stars/dagster-io/dagster?label=Star&style=social"></a>
```

Comment:
how about making these into a new line - it's a bit crowded to have 5 widgets together in one line and separating can make the social ones more explicit

Reply:
took inspiration from https://github.com/facebook/react